packages/dds/tree/src/test/feature-libraries/sequence-field/utils.ts

```diff
@@ -60,7 +60,7 @@ export function rebase(change: TestChangeset, base: TaggedChange<TestChangeset>)
 	if (moveEffects.isInvalidated) {
 		moveEffects.reset();
 		rebasedChange = SF.amendRebase(rebasedChange, base, idAllocator, moveEffects);
-		// assert(!moveEffects.isInvalidated, "Rebase should not need more than one amend pass");
+		assert(!moveEffects.isInvalidated, "Rebase should not need more than one amend pass");
```

Comment:
Is re-enabling these asserts sufficient to test this functionality?

Reply:
I'm not sure I understand the question, but these asserts would have failed on many tests before this PR, and reenabling them tests the goal of my changes.